.github/workflows/pr.yaml

```diff
@@ -38,7 +38,7 @@ jobs:
       build_type: pull-request
       node_type: "cpu8"
       arch: "amd64"
-      container_image: "rapidsai/ci:latest"
+      container_image: "rapidsai/ci:cuda11.8.0-ubuntu20.04-py3.10"
```

Comment:
Previously, the `latest` image pointed to Ubuntu 22.04. Let's keep that for consistency with past results.
```suggestion
      container_image: "rapidsai/ci:cuda11.8.0-ubuntu22.04-py3.10"
```

Reply:
Ref: https://github.com/rapidsai/ci-imgs/pull/56/